README.md

```diff
@@ -0,0 +1,129 @@
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>Capstone</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [🤝 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [Capstone] <a name="about-project"></a>
+
+**[Capstone]** is an exercise in week five to build with HTML and CSS to demostrate the usage of Flexbox, images and backgrounds and Figma. For this capstone project. We built a dev con website that showcases developer events.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>HTML</summary>
+  <ul>
+    <li><a href="https://www.html.com/">HTML</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>CSS</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>JAVASCRIPT</summary>
+  <ul>
+    <li><a href="https://www.javascript.com/">JavaScript</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Has an a top bar with a hambugger button and My Logo**
+- **Responsive on both desktop and mobile**
+- **Creates speakers in the speaker section dynamically**
+- **Has a show more that shows more speakers on mobile**
+- **Has two pages; about and the home page**
+- **Has previous events in the about page**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://veronica365.github.io/capstone/)
+
+## 🚀 Project documentation <a name="project-outline"></a>
+
+- [Project documentation](https://www.loom.com/share/772bcd282d4c4abd8d32fb1738903f9c)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- A [browser](https://www.google.com/search?q=what+is+a+browser&oq=what+is+a+browser&aqs=chrome..69i57.2748j0j1&sourceid=chrome&ie=UTF-8) of your choice
+- That you have set up Git on you desired computer
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd your-desired-folder
+  git clone git@github.com:veronica365/capstone.git
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+- open the repo folder `capstone`
+- Rightclick on the index.html file and select open in the browser
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
```

Comment:
- [ OPTIONAL ] Please since you are the only contributor to this project you can singularize the `Authors` section to `Author` and replace `Author1` with your name. :smile: